.github/workflows/build_main_documentation.yml

```diff
@@ -65,6 +70,15 @@ jobs:
           cd build_dir/optimum
           wget https://raw.githubusercontent.com/huggingface/doc-build/main/optimum/_versions.yml
 
+      - name: Make Graphcore documentation
+        run: |
+          cd optimum-graphcore
+          git fetch origin
```

Comment:
These 2 lines should be removed once https://github.com/huggingface/optimum-graphcore/pull/126 is merged